### ISSUE TITLE
Export can fail to create because generated filename is too long.

### ISSUE THREAD

**Issue by westi** [wp-cli/export-command#38]
Because we use `get_bloginfo( 'name' );` to [generate the filename](https://github.com/wp-cli/export-command/blob/master/src/Export_Command.php#L200) for the export and this is user supplied data we can end up generating a very long filename which then we can't save to disk.

We should consider limiting the info we pull from here using `substr` or using something different.

**Comment by schlessera** [comment]
Yes, I agree, just taking on a potentially unlimited string as a filename should be considered a bug.

Are you up to producing a pull request for the above?
